manager/backingimage.go

```diff
@@ -77,62 +74,6 @@ func (m *VolumeManager) CreateBackingImage(name, checksum, sourceType string, pa
 		return nil, fmt.Errorf("invalid name %v", name)
 	}
```

Comment:
Do we still need this? it seems we already cover in the mutator and validator.

Reply:
Yep, should remove this part.